### PR TITLE
adds note about ad cs

### LINKED ISSUE #16745: [Docs] Desktop Access LDAPS CA
## Details

Re: [Getting Started With Desktop Access](https://goteleport.com/docs/desktop-access/getting-started/?scope=enterprise#step-57-export-your-ldap-ca-certificate)

In [step 5](https://goteleport.com/docs/desktop-access/getting-started/?scope=enterprise#step-57-export-your-ldap-ca-certificate) we export an existing key from the Certification Authority tool. In my testing on Windows 2019 I wasn't able to get any LDAPS (only LDAP) connection until I had created a "Local Computer" CA with KDC Authentication using the Microsoft Management Console (`mmc`) tool.

I assume most customers will already have an appropriate certificate solution for LDAPS, but I'm not sure if they will get the right one exporting from Certification Authority versus MMC.
 
### Category
- Improve Existing

### PR DESCRIPTION
**Pull request by ibeckermayer**
closes https://github.com/gravitational/teleport/issues/16745

### CODE REVIEW

**Review by ptgott — APPROVED** [review]

File: docs/pages/desktop-access/getting-started.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ Desktop Access and log into a Windows desktop from that domain.
 This guide requires you to have:
 
 - An Active Directory domain, configured for LDAPS (Teleport requires an
-  encrypted LDAP connection)
+  encrypted LDAP connection). Typically this means installing [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:17]: I'm not sure this is a meaningful addition.

Installing AD CS is part of the setup guide, not a prerequisite.

Comment (ibeckermayer) [reply docs/pages/desktop-access/getting-started.mdx:17]: What's the setup guide? I only found it [in our troubleshooting documentation](https://github.com/gravitational/teleport/blob/82b001f9c33bf8d94d028d9e3e4963afc6aafb82/docs/pages/desktop-access/troubleshooting.mdx?plain=1#L184)

Comment (zmb3) [reply docs/pages/desktop-access/getting-started.mdx:17]: 🤦 you're right, ignore me. I must have been thinking about the README, not docs.

**Review by zmb3 — APPROVED** [review]

File: docs/pages/desktop-access/getting-started.mdx (hunk 1 above)
Comment (alexfornuto) [reply docs/pages/desktop-access/getting-started.mdx:17]:
```suggestion
  encrypted LDAP connection). Typically this means installing
  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
```

**Review by alexfornuto — APPROVED** [review]
👍  with one nit for line breaks.

**Pull request merged**